xclim/indices/_threshold.py

```diff
@@ -2059,6 +2060,76 @@ def windy_days(
     return out
 
 
+@declare_units(ua="[speed]")
+def jetstream_metric_woolings(
+    ua: xarray.DataArray,
+) -> xarray.DataArray:
+    """Strength and latitude of jetstream.
+
+    Identify latitude and strength of maximum smoothed zonal windspeed in the region 15-75&deg;N, -60-0&deg;E.
+
+    Parameters
+    ----------
+    ua : xarray.DataArray
+      u-component wind velocity at between 750 and 950 hPa.
+
+    Returns
+    -------
+    xarray.DataArray
+      Daily time series of latitude of jetstream.
+    xarray.DataArray
+      Daily time series of strength of jetstream.
+
+    References
+    ----------
+    .. [woollings2010] Woollings, T., Hannachi, A., & Hoskins, B. (2010). Variability of the North Atlantic eddy‐driven jet stream. Quarterly Journal of the Royal Meteorological Society, 136(649), 856-868.
+
+    """
+
+    # select only relevant hPa levels, compute zonal mean windspeed
+    pmin = convert_units_to("750 hPa", ua.plev)
+    pmax = convert_units_to("950 hPa", ua.plev)
```

Comment:
This relies on pressure levels being named `plev`. I suggest we use cf-xarray to identify the pressure coordinate. 
Off the top of my head, it would look like
```
import cf_xarray
...
pmin = convert_units_to("750 hPa", ua.cf["pressure"])
```

I don't know if we want to bring cf-xarray into xclim within this PR or another separate one though. @aulemahal  Thoughts ?